src/Reflex/Dom/SemanticUI/Button.hs

```diff
@@ -124,14 +137,16 @@ uiButtonAttrs UiButton{..} = T.unwords $ catMaybes
 -- provided by other functions such as 'uiButtonAnimated'.
 uiButton
     :: MonadWidget t m
-    => Dynamic t UiButton
+    => UiButtonType
```

Comment:
Instead of changing this type signature, can you put `UiButtonType` inside the `UiButton` type instead?